VideoLocker/res/values/styles.xml

```diff
@@ -83,15 +83,18 @@
     <style name="last_access_bar_style">
         <item name="android:layout_width">match_parent</item>
         <item name="android:layout_height">75dp</item>
-        <item name="android:background">@color/cyan_1</item>
+        <item name="android:background">@color/edx_brand_primary_x_light</item>
         <item name="android:visibility">gone</item>
     </style>
 
-    <style name="download_in_progress_bar_style">
-        <item name="android:layout_width">match_parent</item>
+    <style name="download_in_progress_bar_style" parent="last_access_bar_style">
         <item name="android:layout_height">45dp</item>
-        <item name="android:background">@color/white</item>
-        <item name="android:visibility">gone</item>
+    </style>
+
+    <style name="separator_style">
```

Comment:
This should be named `course_header_separator` or something similar, as there are multiple separators of different colors. Perhaps there should be a base `separator` style with just the dimensions defined, which will be extended by all the implementations defining the color.